node/package.json

```diff
@@ -1,3 +1,4 @@
 {
+  "main": "lib",
```

Comment:
Nesting the entire `node`/`native` builds in a `lib` directory so it's easier to perform a cleanup.